### PR TITLE
[FIX] point_of_sale fix timezone date for cash return account move

### PR DESCRIPTION
**Pull request by cvinh**
Description of the issue/feature this PR addresses:
- Setup the user with timezone GMT-10
- Open a pos session in the afternoon (in GMT the date is +1)
- Pay a pos order in cash and give change back to the customer
Current behavior before PR:
- In the Cash statement, the return move date is at GMT
Desired behavior after PR is merged:
- In the Cash staetment, the return move date should be in GMT-10

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by celm1990** [comment]
Related to 2157540768112107a3c68ed28ce429572e28fce5 or is same?

**Comment by invitu** [comment]
@celm1990 your commit was not applied to the returning cash.

File: addons/point_of_sale/models/pos_order.py [hunk 1]
```diff
@@ -143,7 +145,7 @@ def _process_order(self, pos_order):
                 cash_journal_id = cash_journal[0].id
             order.add_payment({
                 'amount': -pos_order['amount_return'],
-                'payment_date': fields.Datetime.now(),
+                'payment_date': payment_date,
```
Comment (celm1990) [inline addons/point_of_sale/models/pos_order.py]: Is more simply, only  fields.Date.context_today(self) instead fields.Datetime.now().

**Comment by invitu** [comment]
@celm1990 is that how you suggested ? (i did not test)

**Comment by celm1990** [comment]
Yes, is more clarified, tanks

File: addons/point_of_sale/models/pos_order.py [hunk 2]
```diff
@@ -1093,10 +1093,10 @@ def get_sale_details(self, date_start=False, date_stop=False, configs=False):
                 SELECT aj.name, sum(amount) total
                 FROM account_bank_statement_line AS absl,
                      account_bank_statement AS abs,
-                     account_journal AS aj
```
Comment (mart-e) [inline addons/point_of_sale/models/pos_order.py]: please avoid cosmetic changes, keep the diff minimal

**Comment by mart-e** [comment]
Merged at a19d2c98f4c, thanks

**Pull request closed without merging**